src/components/notifications/ConfigurationTab.tsx

```diff
@@ -536,15 +536,15 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
     render() {
         if (this.state.view === ViewType.LOADING) {
             return (
-                <div style={{ height: 'calc(100vh - 172px)' }}>
+                <div className="dc__height_reduce_to_172">
```

Comment:
this should be like `dc__height-reduce-172`

Reply:
done